### PR TITLE
docs: Add Skip Wizard Steps KB

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: knowledge-base/wizard-skip-step.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Skip a Wizard Step
```
Comment (dimodi) [inline knowledge-base/wizard-skip-step.md:2]:
```suggestion
title: How to Skip a Wizard Step
```

File: knowledge-base/wizard-skip-step.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+description: How to skip a TelerikWizard step, based on custom condition from another step?
```
Comment (dimodi) [inline knowledge-base/wizard-skip-step.md:3]:
```suggestion
description: How to skip a step in the TelerikWizard, based on custom condition from another step?
```

File: knowledge-base/wizard-skip-step.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+tags: wizard, skip, step
+ticketid: 1541661
+res_type: kb
+---
+
+## Description
+
+How to skip one or more Wizard steps, based on the results of another step?
+
+The [Wizard flow is linear]({%slug wizard-structure-stepper%}#linear-flow), so I can't disable steps, but I need to skip an unneeded step programmatically, based on a custom condition in a previous step.
```
Comment (dimodi) [inline knowledge-base/wizard-skip-step.md:17]:
```suggestion
The [Wizard flow is set to linear]({%slug wizard-structure-stepper%}#linear-flow), so I can't disable steps, but I need to skip an unneeded step programmatically, based on a custom condition in a previous step.
```

File: knowledge-base/wizard-skip-step.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    <WizardSteps>
+        <WizardStep>
+            <Content>
+                <p>This is Step 1.</p>
+            </Content>
+        </WizardStep>
+        <WizardStep OnChange="@OnStep1Change">
+            <Content>
+                <p>This is Step 2.</p>
+                <label><TelerikCheckBox @bind-Value="@SkipStep2" /> Skip Step 3</label>
```
Comment (ntacheva) [inline knowledge-base/wizard-skip-step.md:43]:
```suggestion
                <label><TelerikCheckBox @bind-Value="@SkipStep3" /> Skip Step 3</label>
```
Shall we name the variable SkipStep**3** as it is a bit confusing like so

File: knowledge-base/wizard-skip-step.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+            <Content>
+                <p>This is Step 4.</p>
+            </Content>
+        </WizardStep>
+    </WizardSteps>
+</TelerikWizard>
+
+@code {
+    int WizardStep { get; set; } = 1;
+    bool SkipStep2 { get; set; } = true;
```
Comment (ntacheva) [inline knowledge-base/wizard-skip-step.md:61]:
```suggestion
    bool SkipStep3 { get; set; } = true;
```

File: knowledge-base/wizard-skip-step.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
+---
+title: How to Skip a Wizard Step
+description: How to skip a step in the TelerikWizard, based on custom condition from another step?
+type: how-to
+page_title: How to Skip a Wizard Step Programmatically
+slug: wizard-skip-step
+position:
+tags: wizard, skip, step
+ticketid: 1541661
```
Comment (dimodi) [inline knowledge-base/wizard-skip-step.md:9]:
```suggestion
ticketid: 1541661, 1544088
```

File: knowledge-base/wizard-skip-step.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+The [Wizard flow is set to linear]({%slug wizard-structure-stepper%}#linear-flow), so I can't disable steps, but I need to skip an unneeded step programmatically, based on a custom condition in a previous step.
+
+## Solution
+
+1. Subscribe to the [`OnChange`]({%slug wizard-events%}#onchange) event of the originating step.
+1. Check `args.TargetIndex` to find out if the user is navigating in the desired direction.
+1. Set `args.IsCancelled` to `true` to cancel navigation to the next step.
+1. Set the Wizard `Value` to the desired step.
+
```
Comment (dimodi) [inline knowledge-base/wizard-skip-step.md:25]:
```suggestion

Note that Wizard step indexes are zero-based for the developer and 1-based for the end user.

```

**Review by ntacheva — APPROVED** [review]

**Pull request merged**